ground/uavobjgenerator/uavobjectparser.cpp

```diff
@@ -404,6 +404,12 @@ QString UAVObjectParser::parseXML(QString& xml, QString& filename)
         // Calculate size
         calculateSize(info);
 
+        // Buffer on flight side is 256 bytes
+        // Additional overhead is 2+4+2+2+1+1=12
+        if(info->numBytes > 244)

```

Comment:
That looks good (although I never know if ```#define``` or ```const``` is preferable in this situation). If you get around to it, it would be good to comment on where the ```2+4+2+2+1+1``` is coming from, although it's sufficiently clear now what it's related to.